model/BUILD.bazel

```diff
@@ -206,6 +219,7 @@ android_library(
     visibility = ["//visibility:public"],
     exports = [
         ":arguments_java_proto_lite",
+        ":feedback_reporting_java_proto_lite",
```

Comment:
Prefer not exporting it--we actually want to add these as direct dependencies wherever they're used rather than having a single library to depend on.

Reply:
Done